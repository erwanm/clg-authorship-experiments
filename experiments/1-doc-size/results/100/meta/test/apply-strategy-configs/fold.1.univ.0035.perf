0.312592	0.558200	0.560000
