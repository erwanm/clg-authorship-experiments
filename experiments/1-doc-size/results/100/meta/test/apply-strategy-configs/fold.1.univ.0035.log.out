apply-config.sh: applying config '/tmp/100/process/selected-meta-configs/fold.1.0013.model/strategy-configs/fold.1.univ.0035' to cases '/home/moreaue/dump/expe1//results/100/meta/test/apply-strategy-configs/only-cases.tmp.rK3mDH15'; results written to '/home/moreaue/dump/expe1//results/100/meta/test/apply-strategy-configs/fold.1.univ.0035'
apply-config.sh: done.
